kibble.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "core-template",
   "version": "0.0.1",
-  "siteUrl": "https://staging-abccinemas.screenplus.co",
+  "siteUrl": "https://abccinemas.screenplus.co",
```

Comment:
Assuming you committed this by accident? lol

Reply:
Yea will change that